lightning-background-processor/src/lib.rs

```diff
@@ -931,6 +933,10 @@ impl BackgroundProcessor {
 	/// [`Persister::persist_manager`] returns an error. In case of an error, the error is retrieved by calling
 	/// either [`join`] or [`stop`].
 	///
+	/// This method takes a [`BackgroundProcessorConfig`] object that contains all necessary components for
```

Comment:
Perhaps some of the docs for `start` are now better positioned in the config object docs?